include/no_os_tdm.h

```diff
@@ -109,6 +117,12 @@ struct no_os_tdm_platform_ops {
 	int32_t (*tdm_ops_read)(struct no_os_tdm_desc *, void *, uint16_t);
 	/** TDM write operation function pointer */
 	int32_t (*tdm_ops_write)(struct no_os_tdm_desc *, void *, uint16_t);
+	/** Pause TDM DMA transfer */
+	int32_t (*tdm_ops_pause_dma)(struct no_os_tdm_desc *);
```

Comment:
I'd drop the `_dma` suffix, it is unnecessary. Technically the implementation of the non-blocking TDM read/write could be done with just interrupts and memcpy's instead of DMA and this generic API needs to cover that case too.

Reply:
Noted. The "_dma" suffix has been removed everywhere